README.md

```diff
@@ -42,12 +42,13 @@ compiler warnings that break the build earlier.
 
 Once you have your dependencies, you can run cmake and then build:
 
-1. Check out this repository.
-1. Create a build directory (you can have the build directory anywhere you
-   like). There is currently a directory in the source tree called build, but you should not use it. See [#3098](https://github.com/apple/foundationdb/issues/3098)
-1. `cd <PATH_TO_BUILD_DIRECTORY>`
-1. `cmake -G Ninja <PATH_TO_FOUNDATIONDB_DIRECTORY>`
-1. `ninja # If this crashes it probably ran out of memory. Try ninja -j1`
+```bash
+git clone https://github.com/apple/foundationdb.git
+cd foundationdb
+mkdir build && cd build
```

Comment:
We don't want to recommend this. I am aware that this is how people often build `cmake` project, but it is generally not a good practice.

Reply:
I wasn't aware of that bad practice. Should I restore the README in his original state and keep the `.gitignore` change?